packages/connection/src/Manager.php

```diff
@@ -533,6 +558,22 @@ public function get_connected_user_data( $user_id = null ) {
 		return false;
 	}
 
+	/**
+	 * Returns a user object of the connection owner.
+	 *
+	 * @return object|false False if no connection owner found.
+	 */
+	public function get_connection_owner() {
+		$user_token = $this->get_access_token( JETPACK_MASTER_USER );
+
+		$connection_owner = false;
+		if ( $user_token && is_object( $user_token ) && isset( $user_token->external_user_id ) ) {
+			$connection_owner = get_userdata( $user_token->external_user_id );
```

Comment:
If the user is missing, this will break. I will update this to do the same as it's done in https://github.com/Automattic/jetpack/pull/13323 by returning ID only once it's merged.

Reply:
#13323 has now been merged.